kqueen_ui/utils/logger_config.yml

```diff
@@ -49,7 +49,7 @@ root:
 
 loggers:
     kqueen_ui:
-        level: INFO
+        level: DEBUG
```

Comment:
Please, leave INFO level as default

Reply:
logging level increased to avoid uncatching issues
i mean handlers have debug level by default, and if  logger have INFO level, then all cases related to "logger.debug(***" doesnot catched, so by default we should set DEBUG level or refactor all existing loggers to INFO, but imo its bad case
@katyafervent